code/modules/mob/living/silicon/ai/ai.dm

```diff
@@ -528,22 +529,38 @@ var/list/ai_verbs_default = list(
 	if(check_unable())
 		return
 
-	var/input
+	var/received //var/input caused too many problems, because there's also the input proc.
 	var/choice = alert("Would you like to select a hologram based on a (visible) crew member, switch to unique avatar, or load your character from your character slot?",,"Crew Member","Unique","My Character")
 
 	switch(choice)
 		if("Crew Member") //A seeable crew member (or a dog)
-			var/list/targets = trackable_mobs()
+			var/list/targets = list()
+			for(var/mob/living/M in mob_list)
+				if(!M)
+					continue
+				if(!M.client) //This *will* keep SSD players from showing up on the list, but if this wasn't here, there would be a runtime if you tried to imitate an SSD mob.

```

Comment:
If you are ignoring clientless mobs, use the player_list instead, which only includes mobs with clients. It will be faster.